### PR TITLE
feat: removes label based automatic publishing

### PR DESCRIPTION
**Pull request by mathewmeconry**
## Description

Removes the label-based automatic publishing feature because it could be problematic in the monorepo.

Task: DOPS-299

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] I have selected the correct base branch.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [x] Any dependent changes have been merged and published in downstream modules.
- [x] I ran all tests with success and extended them if necessary.
- [ ] I have updated the ``CHANGELOG.md`` file in the root folder of the package after the [UPCOMING] title and before the latest version.
- [ ] I have tested my code on the test network.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,31 +2,24 @@
...
 
-To handle certain automated tasks, you can add labels to your pull requests. Here is an overview of the available labels and their meaning.
+### With labels
 
-### JavaScript tags
+To automtically publish the new version on pull request merge add one or all of these labels:
+| Label Name | Package that gets published |
+| --- | --- |
+| client-publish | `javascript/modules/client/` |
+| common-publish | `javascript/modules/common/` |
```
Comment (brickpop) [inline README.md]:
```suggestion
| Label Name | Package that gets published |
| --- | --- |
| client-publish | `javascript/modules/client/` | `@aragon/sdk-client` |
| common-publish | `javascript/modules/common/` | `@aragon/sdk-common` |
```

File: README.md [hunk 2]
```diff
@@ -2,31 +2,24 @@
...
-| `<component>:patch` | Triggers a new NPM patch version of the corresponding JS package | Don't add any other `<component>:*` label |
-| `<component>:minor` | Triggers a new NPM minor version of the corresponding JS package | Don't add any other `<component>:*` label |
-| `<component>:major` | Triggers a new NPM major version of the corresponding JS package | Don't add any other `<component>:*` label |
+### With tags
 
-The available components are:
+To publish a new version of a subpackage create a new git tag following this schema:  
+`VERSION-LANGUAGE-PACKAGE_FOLDER_NAME`  
+Example to publish a new version of the javascript client module: `0.0.1-javascript-client`.  
+To publish an alpha version just put `-alpha` behind the semver: `0.0.1-alpha-javascript-client`
```
Comment (brickpop) [inline README.md:22]: Is this being implemented by the updated codebase?

Comment (mathewmeconry) [reply README.md:22]: this was already there but not documented

File: README.md (hunk 1 above)
Comment (mathewmeconry) [reply README.md]: done

**Review by brickpop — APPROVED** [review]

**Pull request merged**